playbooks/appsemblerPlaybooks/ficus_basic.yml

```diff
@@ -3,6 +3,7 @@
 # Appsembler basic tier deployment
 
 - include: monitoring.yml
+  when: SKIP_MONITORING is undefined
```

Comment:
@mrgnr As Black Team "PR Czar" for the week, it's my duty to bug people to help move things along :) Any thoughts on this question from TJ and John?  "Should we automatically disable the monitoring playbook from running if we're deploying to a staging server?"

@johnbaldwin  @tkeemon My 2 cents:  yes, we probably should disable by default, but no t  in cases like ISC where staging is truly a content staging server and not just a test server.